cmplibrary/src/main/java/com/sourcepoint/cmplibrary/util/ViewsManager.kt

```diff
@@ -164,14 +168,24 @@ private class ViewsManagerImpl(
                     messSubCat = messSubCat,
                     viewId = cmpViewId
                 )
+                this.webview = newWebView
+                newWebView
             }
         } ?: Either.Left(WebViewCreationException(description = "The activity reference in the ViewManager is null!!!"))
     }
-
     override fun dispose() {
+        webview = null
```

Comment:
this assignment should be present also into the `removeAllViews` and `removeAllViewsExcept` methods